Ginger/GingerCoreNET/Run/RunListenerLib/CenteralizedExecutionLogger/AccountReportEntitiesDataMapping.cs

```diff
@@ -424,15 +424,29 @@ private static void SetRunnerChildCounts(GingerExecutionEngine runner, AccountRe
 
             foreach (BusinessFlow businessFlow in runner.BusinessFlows)
             {
-                ChildExecutableItemsCountActivity = ChildExecutableItemsCountActivity + businessFlow.Activities.Count(x => x.Active && (x.Status == eRunStatus.Passed || x.Status == eRunStatus.FailIgnored || x.Status == eRunStatus.Failed || x.Status == eRunStatus.Blocked));
+                if(IsStart)
+                {
+                    ChildExecutableItemsCountActivity = ChildExecutableItemsCountActivity + businessFlow.Activities.Count(x=> x.Active);
+                }
+                else
+                {
+                    ChildExecutableItemsCountActivity = ChildExecutableItemsCountActivity + businessFlow.Activities.Count(x => x.Active && (x.Status == eRunStatus.Passed || x.Status == eRunStatus.FailIgnored || x.Status == eRunStatus.Failed || x.Status == eRunStatus.Blocked));
```

Comment:
what is different between the calculation of ChildExecutableItemsCountActivity in line 433 and in line 436?

Reply:
line no 433 (ChildExecutableItemsCountActivity) is total activity count and line no 436(ChildExecutedItemsCountActivity) is total executed activity count.